lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1199,16 +1214,29 @@ void addIntelFPGADecorations(
 void addIntelFPGADecorationsForStructMember(
     SPIRVEntry *E, SPIRVWord MemberNumber,
     std::vector<std::pair<Decoration, std::string>> &Decorations) {
+  // FIXME: move this code out of this function, it is not IntelFPGA-specific
```

Comment:
Ditto?

Reply:
I've refactored code related to IntelFPGA-specific decorations and removed this FIXME